packages/radio/src/radio.css

```diff
@@ -20,14 +20,3 @@ governing permissions and limitations under the License.
 :host([disabled]) {
     pointer-events: none;
 }
-
-/**
- * Work around for https://github.com/adobe/spectrum-css/issues/2251
- **/
-:host([dir='rtl']) #button:after {
```

Comment:
Yay this is not needed because the exact same code is coming from the spectrum-radio file!